src/algorithms/tonal/key.cpp

```diff
@@ -367,21 +367,21 @@ void Key::compute() {
 
 
   if (maxMajor > maxMinor && maxMajor > maxOther) {
-    keyIndex = std::roundl(keyIndexMajor *  12 / pcpsize);
+    keyIndex = std::lround(keyIndexMajor *  12 / pcpsize);
     scale = MAJOR;
     max = maxMajor;
     max2 = max2Major;
   }
 
   else if (maxMinor >= maxMajor && maxMinor >= maxOther) {
-    keyIndex = std::roundl(keyIndexMinor * 12 / pcpsize);
+    keyIndex = std::lround(keyIndexMinor * 12 / pcpsize);
     scale = MINOR;
     max = maxMinor;
     max2 = max2Minor;
     }
 
 	else if (maxOther > maxMajor && maxOther > maxMinor) {
-    keyIndex = std::roundl(keyIndexOther * 12 / pcpsize);
+    keyIndex = std::lround(keyIndexOther * 12 / pcpsize);
     scale = MAJMIN;
     max = maxOther;
```

Comment:
`keyIndex` is an int, not `long int`, therefore use `round` instead of `lround`. This will be consistent with code in other algorithms doing similar rounding.

Reply:
By using std::round we are relying on implicit conversion (float to integer), and this conversion, when the value is out of the range of int type, is a UB. Using std::lround() prevent that UB because std::lround with set the floating environemtn error if the value is too big to be represented by long int, and convert a long int with too big value to int is only unspecified (pre C++20) or well-defined (since C++20). 
Anyway the difference is very insignificant. When the value is within the range of int both ways give correct result, and I believe using std::lround is safer.

Reply:
In practice we'll never get too big values in this case. In general I agree which such changes, but they should be systematically applied to the entire code base.